src/templates/runs/paginated-list.twig

```diff
@@ -55,7 +55,7 @@
               </a>
             </td>
             <td>
-                {% set addr = result.meta.url %}
+                {% set addr = result.meta('SERVER.SERVER_NAME') ~ result.meta.url %}
```

Comment:
Won't this be the server that xhgui is running on? That isn't necessarily the same server/host that the profile data was collected from.

Reply:
As I saw show the right domain and not the server one.

Reply:
You're right. I misread the code earlier.